plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcSystemTables.java

```diff
@@ -22,4 +28,25 @@ public TestIcebergOrcSystemTables()
     {
         super(ORC);
     }
+
+    @Test
+    @Override
+    public void testPropertiesTable()
```

Comment:
Can we fix the base test instead of overriding?

Reply:
Sure thing!